ndbench-core/src/main/java/com/netflix/ndbench/core/discovery/AWSLocalClusterDiscovery.java

```diff
@@ -0,0 +1,47 @@
+package com.netflix.ndbench.core.discovery;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AWSLocalClusterDiscovery implements IClusterDiscovery {
+    private static final Logger logger = LoggerFactory.getLogger(LocalClusterDiscovery.class.getName());
+
+    @Override
+    public List<String> getApps() {
+        return Arrays.asList(getLocalhostName());
+    }
+
+    private String getLocalhostName() {
+    	String url = "http://169.254.169.254/latest/meta-data/public-hostname";
```

Comment:
This works great on EC2-Classic instances, but does not work on EC2-VPC instances, so either make this class AWS**Classic**LocalClusterDiscovery or support VPC instances in getLocalhostName() method.